lerna.json

```diff
@@ -4,5 +4,5 @@
     "packages/*"
   ],
   "npmClient": "yarn",
-  "version": "1.1.0"
+  "version": "1.2.3"
```

Comment:
This and the next 3 files have inconsistent versioning.  Is that expected?

Reply:
kind of expected... I didn't know what I was doing while using https://lernajs.io/ but I do now :)  I'll sync these up